doc/install.md

```diff
@@ -0,0 +1,118 @@
+[doc](../README.md) > Installation
+
+This system consists of two elements: station (which is intended to run on a Raspberry Pi, but can be run on any Linux
+box) and a server (which is intended to be run in a place with good uplink connectivity). If you are interested in
+running your own station, you most likely want to deploy just the station and use existing server. Please contact
+someone from the satnogs team and we'll hook you up.
+
+# Station installation
+
+You need to install [Rasbian](https://www.raspbian.org/) on your Raspberry Pi. Please follow any Raspbian installation
+instruction, such as [this one](https://www.raspberrypi.org/documentation/installation/installing-images/). Once
+done, connect to your Pi and do the following as root:
+
+1. **Install necessary dependencies**:
+
+```
+apt update
+apt install python3-minimal git
```

Comment:
We have also other dependencies as noaa-apt, medet, meteor-decode, rtl_fm, sox, convert.

Reply:
Updated. See if the description works for you.